content/applications/sales/sales/invoicing/proforma.rst

```diff
@@ -7,23 +7,24 @@ notes the kind and quantity of goods, their value, and other important informati
 and transportation charges.
 
 Pro-forma invoices are commonly used as preliminary invoices with a quotation. They are also used
-for customs purposes during importation. They differ from a normal invoice in that they are not a
+during importation for customs purposes. They differ from a normal invoice, in that they are *not* a
 demand (or request) for payment.
 
-Activate the feature
-====================
+Configuration
+=============
 
-In order to utilize pro-forma invoices, the *Pro-Forma Invoice* feature **must** be activated. To do
-that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
+In order to utilize pro-forma invoices, the *Pro-Forma Invoice* feature **must** be activated.
+
+To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
 :guilabel:`Quotations & Orders` section, click the checkbox next to :guilabel:`Pro-Forma Invoice`.
 Then, click :guilabel:`Save` to save all changes.
 
 .. image:: proforma/pro-forma-setting.png
    :align: center
    :alt: The Pro-Forma Invoice feature setting in the Odoo Sales application.
 
-Send a pro-forma invoice
-========================
+Send pro-forma invoice
+======================
 
 With the :guilabel:`Pro-Forma Invoice` feature activated, the option to send a pro-forma invoice is
 now available on any quotation or sales order, via the :guilabel:`Send Pro-Forma Invoice` button.
```

Comment:
After some testing it looks like you can't send a pro-forma invoice on a SO/quotation for if you have already sent an invoice for a down payment (or a recurring subscription, so the button will not appear in that case. May be worth a call out. (You can send one for services/event registrations/courses/new subscriptions, so it doesn't seem to be limited to physical/consumable/storable goods)